petpals/messaging/templates/new_message.html

```diff
@@ -0,0 +1,64 @@
+{% extends '_layout.html' %}
+{% block title %}New Message{% endblock %}
+{% set active_page = 'messages' %}
```

Comment:
Incorrect value.

Reply:
keeping active page as messages for navbar purposes